.env.local.default

```
@@ -50,3 +50,7 @@ DREVOPS_DB_DOWNLOAD_SSH_KEY_FILE=$HOME/.ssh/id_rsa
 # GitHub token used to overcome API rate limits or access private repositories.
 # @see https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
 GITHUB_TOKEN=
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)